### ISSUE TITLE
Bug: don't flag first of two for variables as unused?

### ISSUE THREAD

**Issue by helixbass** [helixbass/eslint-plugin-coffee#18]
This is getting flagged by `no-unused-vars`:
```
for v, i in a by f()
  forCompile.push i
```
But since there's no way to avoid defining `v`, it shouldn't get flagged?

**Comment by edemaine** [comment]
For what it's worth, I just ran into this issue in my own code. (Not sure why this didn't trigger when I was using `@fellow/eslint-plugin-coffee`...)

I'm using the obvious workaround:
```coffee
for v, i in a by f() # eslint-disable-line coffee/no-unused-vars
  forCompile.push i
```

**Comment by helixbass** [comment]
@edemaine ok fixed in #54 

Until I do a new beta/full release you could use a dev version which includes this change via:
```
yarn add --dev github:helixbass/eslint-plugin-coffee#eslint-plugin-coffee-v0.1.15-dev.2-gitpkg
```
